### DISCUSSION TITLE
Mockito

### DISCUSSION THREAD

**Discussion by Serrof** [Hipparchus-Math/hipparchus#319] [Ideas]
Hi all,

I know Hipparchus is very light weight when it comes to dependencies but I would really appreciate if we could use Mockito for tests. I cannot emphasize enough how good it is. And it's already used for Orekit. Thoughts?

Cheers,
Romain.

**Comment by axkr** [comment]
As Mockito is only used for tests, it can be used in `<scope>test</scope>` just as JUnit:
- https://github.com/Hipparchus-Math/hipparchus/blob/05c939f1b918c6d92735bc550a49b2088515d92e/hipparchus-core/pom.xml#L46

Because it is not a "regular dependency" I also see no problems.

**Comment by maisonobe** [comment]
I agree with Axel comments, dependencies for tests are perfectly acceptable.

**Comment by Serrof** [comment]
Great. I'll open an issue. Let's see there if we want to do that for the upcoming release
